BankAPILibrary/GiniBankAPILibrary/Sources/GiniBankAPILibrary/Documents/Document.swift

```diff
@@ -135,12 +156,22 @@ extension Document {
         public let document: URL
         public let pages: URL?
         
-        public init(extractions: URL, layout: URL, processed: URL, document: URL, pages: URL?) {
-            self.extractions = extractions
-            self.layout = layout
-            self.processed = processed
-            self.document = document
-            self.pages = pages
+        /**
+         An  initializer for a `Links` structure if you are receiving a customized JSON structure from your proxy backend.
+         For this particular case all links will be pointed to the document's link.
+         
+         - parameter giniAPIDocumentURL: The document's link receive from Gini API. For example "https://pay-api.gini.net/documents/\(documentID)".
```

Comment:
I would add a bit more details and an example link with a fake document id:
```suggestion
         - parameter giniAPIDocumentURL: The document's link received from the Gini API. This must be the same URL that you received in the `Location` header from the Gini API. For example "https://pay-api.gini.net/documents/626626a0-749f-11e2-bfd6-000000000000".
```.